queue_services/business-pay/src/business_pay/resources/pay_filer.py

```diff
@@ -109,7 +109,7 @@ async def worker():
         logger.debug(f"Removed From Queue: no payment info in ce: {str(ce)}")
         return {}, HTTPStatus.OK
 
-    if payment_token.corp_type_code in ["MHR"]:
+    if payment_token.corp_type_code in ["MHR", "BUS", "STRR"]:
```

Comment:
we have `BTR` in sandbox business-pay.  do you know if that's correct?

Reply:
Yes. thats correct